client/imports/account/account.html

```diff
@@ -0,0 +1,54 @@
+<ul id = "toolbar">
+  <li><a class="active" href="./home">Home</a></li>
+  <li><a href="./account">Account</a></li>
+  <li><a href="./coursepage">Courses</a></li>
+</ul>
+
+<style>
+#header {
+    background-color:black;
+    color:white;
+    text-align:center;
+    padding:5px;
+}
+#nav {
```

Comment:
These should not be defined in the first place- we are using Angular Material for a reason.